src/search/landmarks/landmark_status_manager.cc

```diff
@@ -1,171 +1,223 @@
 #include "landmark_status_manager.h"
+#include "util.h"
 
 #include "landmark.h"
 
-#include "../utils/logging.h"
-
 using namespace std;
 
 namespace landmarks {
+static vector<LandmarkNode *> get_goal_landmarks(const LandmarkGraph &graph) {
+    vector<LandmarkNode *> goals;
+    for (auto &node : graph.get_nodes()) {
+        if (node->get_landmark().is_true_in_goal) {
+            goals.push_back(node.get());
+        }
+    }
+    return goals;
+}
+
+static vector<pair<LandmarkNode *, vector<LandmarkNode *>>> get_greedy_necessary_children(
+    const LandmarkGraph &graph) {
+    vector<pair<LandmarkNode *, vector<LandmarkNode *>>> orderings;
+    for (auto &node : graph.get_nodes()) {
+        vector<LandmarkNode *> greedy_necessary_children{};
```

Comment:
We don't elsewhere call the default constructor explicitly when this is also what happens implicitly without the {}.